services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -4401,6 +4401,16 @@ public void updateAdminOperationProtocolVersion(String clusterName, Long adminOp
         .updateAdminOperationProtocolVersion(clusterName, adminOperationProtocolVersion);
   }
 
+  @Override
+  public Map<String, Long> getAdminOperationVersionFromControllers(String clusterName) {
+    return getVeniceHelixAdmin().getAdminOperationVersionFromControllers(clusterName);
```

Comment:
BTW, where is the logic that parent also send requests to child controllers in all regions?

Reply:
We will send requests to child controllers in all regions in the next PR where we define the service (with threadpool and task).